packages/SidePanel/src/components/FocusLock/FocusLock.js

```diff
@@ -1,37 +1,8 @@
-/* eslint-disable react/require-default-props */
 import React from "react";
-import PropTypes from "prop-types";
 import FocusLockLibrary from "react-focus-lock";
+import { FocusPropTypes } from "@paprika/helpers/lib/customPropTypes";
 
-const propTypes = {
```

Comment:
this will be shared by at least three focusLock components across modal, sidepanel and takeover